admin/services/data-access/pupil-restart.data.service.js

```diff
@@ -46,10 +54,104 @@ pupilRestartDataService.findLatest = async function (options) {
 
 /**
  * Get all the restart codes documents
+ * @deprecated Please use sqlFindRestartCodes instead
  * @return {Promise.<{Object}>}
  */
 pupilRestartDataService.getRestartCodes = async () => {
   return RestartCode.find().lean().exec()
 }
 
+/** SQL METHODS **/
+
+/**
+ * Create a new pupil restart.
+ * @param {object} data
+ * @return  { insertId: <number>, rowsModified: <number> }
+ */
+pupilRestartDataService.sqlCreate = async (data) => {
+  return sqlService.create(table, data)
+}
+
+/**
+ * Returns number of restarts specified by pupil id
+ * @param pupilId
+ * @return {Promise.<*>}
+ */
+pupilRestartDataService.sqlGetNumberOfRestartsByPupil = async function (pupilId) {
```

Comment:
The naming policy is to use `sqlFind` so this should be `sqlFindNumberOfRestartsByPupil`  - please can you change it?

Reply:
@jon-shipley updated